src/main/java/step5/domain/Cars.java

```diff
@@ -0,0 +1,54 @@
+package step5.domain;
+
+import java.util.ArrayList;
+import java.util.Collections;
+import java.util.List;
+import java.util.stream.Collectors;
+
+public class Cars {
+    private final List<Car> carList;
+
+    public Cars(List<String> carNames) {
+        this.carList = create(carNames);
+    }
+
+    // 테스트 코드를 위한 생성자 (더미 매개변수를 받아서 해도 괜찮은 건지 모르겠습니다  ㅠ)
+    public Cars(List<Car> cars, int dummy) {
+        this.carList = cars;
+    }
+
+    public List<Car> getCarList() {
+        return carList;
+    }
+
+    private List<Car> create(List<String> carNames) {
+        List<Car> cars = new ArrayList<>();
+        for (int i = 0; i < carNames.size(); ++i) {
+            cars.add(new Car(carNames.get(i)));
+        }
+
+        return cars;
+    }
+
+    public void move() {
+        for (int i = 0; i < carList.size(); ++i) {
+            Car car = carList.get(i);
+            car.move(new RandomMovingStrategy());
+        }
+
+        System.out.println();
+    }
+
+    public String findWinner() {
+        Position position = maxDistance();
+        return carList.stream()
+                .filter(car -> car.getPosition().equals(new Position(position.getPosition())))
```

Comment:
getter가 아닌 Car 객체에게 메시지를 보내는 방식으로 구현을 한다면
캡슐화를 지킬 수 있을 것 같아요 :)
```suggestion
                .filter(car -> car.isWinner(position))
```